dm/dm/ctl/master/unlock_ddl_lock.go

```diff
@@ -35,6 +34,9 @@ func NewUnlockDDLLockCmd() *cobra.Command {
 	}
 	cmd.Flags().StringP("owner", "o", "", "source to replace the default owner")
 	cmd.Flags().BoolP("force-remove", "f", false, "force to remove DDL lock")
+	cmd.Flags().String("action", "skip", "whether to skip or execute ddls")
```

Comment:
need to tell user action can be choose between skip/exec

Reply:
addressed in https://github.com/pingcap/tiflow/pull/4087/commits/96b2db76737e4e6221b5f0413690e0be2d8a5e15